spec/model/user_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before do
+    @user = User.create(name: 'Halle', photo: 'Photo', bio: 'Ruby Developper', posts_counter: 0)
+    @posts = []
+  end
+
+  it 'Should be present' do
+    @user.name = 'Halle'
+    expect(@user).to be_valid
+  end
+
+  it 'posts_counter must be an integer and >= to zero' do
+    @user.posts_counter = 0
+    expect(@user).to be_valid
+  end
```

Comment:
- [OPTIONAL] Good job so far with the test cases, but since we are aiming for a 100% test coverage, it will be nice to also include negative test cases, that is tests that should evaluate to false, or not valid.